packages/actions/test/unit/storage.test.ts

```diff
@@ -212,8 +200,7 @@ describe("Storage", () => {
                     bucketName,
                     objectName,
                     localPath,
-                    streamChunkSizeInMb.toString(),
-                    presignedUrlExpirationInSeconds
+                    streamChunkSizeInMb
                 )
                 expect(success).to.be.true
```

Comment:
These tests need changing due to multiPartUpload not returning true/false anymore. Please change to instead:

assert.isFulfilled(multiPartUpload(...))